sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -557,15 +561,29 @@ public partial class DataLakeFileAppendOptions
         public DataLakeFileAppendOptions() { }
         public byte[] ContentHash { get { throw null; } set { } }
         public bool? Flush { get { throw null; } set { } }
+        public Azure.Storage.Files.DataLake.Models.LeaseAction? LeaseAction { get { throw null; } set { } }
+        public System.TimeSpan? LeaseDuration { get { throw null; } set { } }
         public string LeaseId { get { throw null; } set { } }
         public System.IProgress<long> ProgressHandler { get { throw null; } set { } }
+        public string ProposedLeaseId { get { throw null; } set { } }
         public Azure.Storage.UploadTransferValidationOptions TransferValidationOptions { get { throw null; } set { } }
     }
     public enum DataLakeFileExpirationOrigin
     {
         CreationTime = 0,
         Now = 1,
     }
+    public partial class DataLakeFileFlushOptions
+    {
+        public DataLakeFileFlushOptions() { }
+        public bool? Close { get { throw null; } set { } }
+        public Azure.Storage.Files.DataLake.Models.DataLakeRequestConditions Conditions { get { throw null; } set { } }
+        public Azure.Storage.Files.DataLake.Models.PathHttpHeaders HttpHeaders { get { throw null; } set { } }
+        public Azure.Storage.Files.DataLake.Models.LeaseAction? LeaseAction { get { throw null; } set { } }
+        public System.TimeSpan? LeaseDuration { get { throw null; } set { } }
+        public string ProposedLeaseId { get { throw null; } set { } }
+        public bool? RetainUncommittedData { get { throw null; } set { } }
+    }
```

Comment:
> I have chosen to re-use DataLakeRequestConditions.LeaseId to specify the LeaseId for Flush-based leasing operations

There is also a `ProposedLeaseId` here. What is the difference? And are the other values in `Conditions` ignored?

Reply:
`ProposedLeaseId` is used with `LeaseAction.Aquire` when acquiring a new lease on flush.  The other `Conditions` are not ignored.